docs/contributing/forem.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Contributing to Forem
+---
+
+# Contributing to Forem
+
+We expect contributors to abide by our underlying
+[code of conduct](https://dev.to/code-of-conduct). All discussions about this
+project must be respectful and harassment-free.
+
+Remember that communication is the lifeblood of any Open Source project. We are
+all working on this together, and we are all benefiting from this software.
+
+It's very easy to misunderstand one another in asynchronous, text-based
+conversations. When in doubt, assume everyone has the best intentions.
+
+If you feel anyone has violated our Code of Conduct, you should anonymously
+contact the team with our [abuse report form](https://dev.to/report-abuse).
+
+### Where to contribute
+
+All [issues](https://github.com/thepracticaldev/dev.to/issues) labeled
+[ready for dev](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev%22)
+and
+[type: bug](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22)
+are up for grabs.
+
+- [good first issue](https://github.com/thepracticaldev/dev.to/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+)
+  issues are meant for newer developers.
+- [difficulty: easy](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+easy%22)
+  issues are usually confined to isolated areas of existing code.
+- [difficulty: medium](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+medium%22)
+  issues can sometimes entail new features and might affect a significant area
+  of the codebase, but aren't overly complex.
+- [difficulty: hard](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+hard%22)
+  issues will typically be far-reaching, and might need architecture decisions
+  during implementation. This label might also denote highly complex issues.
+
+PRs without an associated issue may still be merged, but the core team will
+focus on changes that solve existing issues. We strongly encourage creating an
+issue before working on a PR!
+
+When in doubt, ask a
+[core team member](https://github.com/thepracticaldev/dev.to/#core-team) by
+mentioning us on the issue.
+
+**Documentation** is almost always a great place to start contributing to a new
+project. DEV is an Open Source, community-driven project. Therefore, providing
```

Comment:
```suggestion
project. Forem is an Open Source, community-driven project. Therefore, providing
```
Should we be using Forem here since the heading is nested within the `Contributing to Forem` page?